src/Components/SportsList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState, useContext } from "react";
+import { SearchKeywordContext } from "./SearchKeywordContext";
+import "./SportsList.scss"
+
+function SportsList () {
+  const [data , setData] = useState([]);
+  const {searchKeyword} = useContext(SearchKeywordContext);
+  
+  useEffect(() => {
+    requestSportsList();
+  });
+    
+  async function requestSportsList() {
+    const result = await fetch(`https://raw.githubusercontent.com/dariusk/corpora/master/data/sports/sports.json`);
+    const {sports} = await result.json();
+
+    if (searchKeyword == "") {
+      setData(sports.slice(0, 20));
+    } else if (searchKeyword.length >= 3) {
+      const results = sports.filter((sport) => {
+        return (
+          sport.toLowerCase().startsWith(searchKeyword.toLowerCase())
```

Comment:
This could be written in the following way to search in the whole name:
```suggestion
          sport.toLowerCase().includes(searchKeyword.toLowerCase())
```

Reply:
Nice.